recipes/libblitz/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.10" %}
+
+package:
+  name: libblitz
+  version: {{ version }}
+
+source:
+  fn: blitz-{{ version }}.tar.gz
+  url: http://downloads.sourceforge.net/project/blitz/blitz/Blitz%2B%2B%20{{ version }}/blitz-{{ version }}.tar.gz
+  md5: 66268b92bda923735f2e3afc87dcb58a
+
+build:
+  number: 0
+  skip: true   # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+
+test:
+  commands:
+    - test -d $PREFIX/include/blitz  # [unix]
+
+about:
+  home: https://sourceforge.net/projects/blitz
+  license: LGPL 3
+  summary: A C++ class library for scientific computing which provides performance on par with Fortran 77/90
+
+extra:
+  recipe-maintainers:
+    - 183amir
+    - bioidiap
```

Comment:
Can I name the organisation in a comment in front of my name?

On Wed, Mar 30, 2016, 4:45 PM jakirkham <notifications@github.com> wrote:

> In recipes/libblitz/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/235#discussion_r57900908>
> :
>
> > +    - automake
> > +    - libtool
> > +
> > +test:
> > +  commands:
> > +    - test -d $PREFIX/include/blitz  # [unix]
> > +
> > +about:
> > +  home: https://sourceforge.net/projects/blitz
> > +  license: LGPL 3
> > +  summary: A C++ class library for scientific computing which provides performance on par with Fortran 77/90
> > +
> > +extra:
> > +  recipe-maintainers:
> > +    - 183amir
> > +    - bioidiap
>
> After some brief discussion, we decided we don't want orgs listed only
> relevant users. So, could you please remove the org?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/235/files/c4a062e4eab1c8c525bf2e215a25d4ba51adf9c1#r57900908>
>